Testing/ORKTest/ORKTest/ORKESerialization.m

```diff
@@ -833,6 +833,12 @@ @implementation ORKESerializer
            PROPERTY(outputVolume, NSNumber, NSObject, NO, nil, nil),
            PROPERTY(samples, ORKToneAudiometrySample, NSArray, NO, nil, nil),
            })),
+   ENTRY(ORKReactionTimeResult,
+         nil,
+         (@{
+            PROPERTY(timestamp, NSNumber, NSObject, NO, nil, nil),
+            PROPERTY(fileResult, ORKResult, NSObject, NO, nil, nil)
```

Comment:
writeAfterInit only needs to be YES if an initializer block is specified. If there's no init block, all parameters are *assumed* to be writeAfterInit since they can't be written in the initializer.